docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Overview
+type: Getting Started
+---
+
+This set of Getting Started guides is an end-to-end scenario that will walk you through all major Kyma functionalities and show its:
+- **Integration and connectivity** feature brought in by [Application Connector](https://kyma-project.io/docs/components/application-connector/). With Kyma, you can connect external applications and expose their API and events on the cluster.
+- **Extensibility** feature provided by [Service Catalog](https://kyma-project.io/docs/components/service-catalog/). You can use its built-in portfolio of external services in your microservices and Functions.
+- **Application runtime** feature supported by [Serverless](https://kyma-project.io/docs/components/serverless/) where you can build microservices and Functions to interact with external services and applications to perform a given business logic.
+
+With these features in mind, we will:
+
+1. Create a microservice that can expose HTTP endpoints and receive a specific type of events from external applications. It has built-in in-memory storage for storing event details, but it can also work with the external Redis storage.
+2. Provision the Redis service through the Service Catalog and use it as an alternative database for the microservice.
+3. Connect am external mock application as an addon to Kyma. We will use it to send events to the microservice with order delivery details.
+4. Create a Function with the logic similar to that of the microservice. You will see how it can also be triggered by order events from the mock application, and save the order details in the external storage.
+
+> **CAUTION:** These guides refer to a sample `orders-service` application deployed on Kyma as a `microservice`. This way it can be easily distinguished from the external Commerce mock that represents an external monolithic application connected to Kyma. In Kyma docs, they are referred to as `application` and `Application` respectively.
+
+All guides, whenever possible, demonstrate the steps from both kubectl and UI.
```

Comment:
```suggestion
All guides, whenever possible, demonstrate the steps in both kubectl and the Console UI.
```

Technically, kubectl (or the terminal) is also a kind of user interface, just not a graphical one, so us calling it `UI` is just a mental shortcut.

Reply:
Yes, I get your point. I just thought that is a mental shortcut that would be clear to everyone but if you think it might not, I can change it to what it was (as I see you also updated the tabs in docs this way).